faust/assignor/partition_assignor.py

```diff
@@ -284,6 +287,25 @@ def _perform_assignment(
         )
         return res
 
+    def _global_table_standby_assignments(
+            self,
+            assignments: dict) -> dict:
```

Comment:
Thanks! There does seem to be some more issues with Global tables. Will keep testing. 
And will change the signature.

Reply:
I implemented Mapping[] as you mentioned, but flakes threw an error saying it should be Dict[str, ClientAssignment] instead of Mapping[str, ClientAssignment]. Should I change it to Dict[str, ClientAssignment]?